artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -1002,43 +1003,51 @@ private void applyAddressesAndQueues(HashMap<String, String> filters) {
    private void performAutoTune(HashMap<String, String> filters, JournalType journalType, File dataFolder) {
       if (noAutoTune) {
          filters.put("${journal-buffer.settings}", "");
+         filters.put("${page-sync.settings}", "");
       } else {
          try {
             int writes = 250;
             System.out.println("");
             System.out.println("Auto tuning journal ...");
 
+            long time = SyncCalculation.syncTest(dataFolder, 4096, writes, 5, verbose, !noJournalSync, false, "journal-test.tmp", ActiveMQDefaultConfiguration.getDefaultJournalMaxIoAio(), journalType);
+            long nanoseconds = SyncCalculation.toNanos(time, writes, verbose);
+            double writesPerMillisecond = (double) writes / (double) time;
+            String writesPerMillisecondStr = new DecimalFormat("###.##").format(writesPerMillisecond);
```

Comment:
If AIO or MAPPED, we should run another test using `NIOSequentialFile` to apply to page-sync-timeout.